lib/composites/alert/index.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+import delegate from 'delegate';
+import open from '../../commons/open';
+import sizer from '../../commons/sizer';
+import trapFocus from '../../commons/trap-focus';
+
+const debug = require('debug')('dqpl:composites:alert');
+
+module.exports = () => {
+  /**
+   * Handle clicks on triggers
+   */
+  delegate(document.body, '[data-alert-id]', 'click', (e) => {
```

Comment:
this exact same delegated click listener on `"[data-alert-id]"` elements is in _composites/modals/index.js_. This may cause unintended behavior on pages that have both a modal and an alert. Once again, let's move this code into commons (since code in _commons_ that applies to both this and the modal is stacking up, I think we should have a _commons/dialogs/_ that houses this, open, close, aria, and trap-focus)